lms/templates/courseware/search.html

```diff
@@ -0,0 +1,115 @@
+<%page expression_filter="h"/>
```

Comment:
This is a new file in edx. I can copy it over to FT theme but I suppose edx core should have it.